Google.Api.Gax.Grpc.Gcp/GcpCallInvokerPool.cs

```diff
@@ -166,12 +126,12 @@ public Key(ServiceEndpoint endpoint, List<ChannelOption> options)
             public override int GetHashCode() =>
                 EqualityHelpers.CombineHashCodes(
                     Endpoint.GetHashCode(),
-                    EqualityHelpers.GetListHashCode(Options, ChannelOptionComparer.Instance));
+                    EqualityHelpers.GetListHashCode(Options, EqualityComparer<ChannelOption>.Default));
```

Comment:
Isn't `EqualityComparer<ChannelOption>.Default` the default? So specifying it here (and a few lines down) isn't necessary?

Reply:
EqualityListHelpers.GetListHashCode doesn't have any default :)

We could remove it from SequenceEqual though.

Reply:
(And I'll do so now)